sensor_msgs_py/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>sensor_msgs_py</name>
+  <version>2.1.0</version>
+  <description>A package for easy creation and reading of PointCloud2 messages in Python.</description>
+  <maintainer email="sebastian.grans@gmail.com">Sebastian Grans</maintainer>
+  <license>BSD</license>
```

Comment:
It mostly only makes sense to have the maintainers of the repository be the maintainers of the sub-packages.  Thus, I suggest you change this to the following:

```suggestion
  <maintainer email="michael@openrobotics.org">Michael Carroll</maintainer>
  <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
  <license>BSD</license>
  <author email="sebastian.grans@gmail.com">Sebastian Grans</author>
```

Reply:
Done! :)